README.md

```diff
@@ -1,2 +1,90 @@
 # CapstoneJS-API-Webapp
 This is a project that involves the creation of a Webapp consumming an API
+
+
+# CapstoneJS-API-Webapp
+
+> This is a project that involves the creation of a Webapp consumming two APIs services.
+
+![Desktop_image](./desktop_ver.png) 
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML5
+- CSS3
+- JavaScript
+- Bootstrap
+- Linters
+- API: Pokemon TCG
+- API: Involvement Microverse
+## Live Demo
+
+If you like, you can see a live demo of the site here:
+
+[Live Demo Link](https://angelacuahutle.github.io/CapstoneJS-API-Webapp/)
```

Comment:
- [ ] Live demo link doesn't work, Please update this field.

Reply:
Thanks for the feedback @sinansevgi. The amends is made on commit f7d9811. Unfortunately, as you can see I'm not the owner of the repo and I'm still trying to reach my partner in order to enable the github page configuration, but without any success, please if you can forget this part. I promise that as soon as my partner gives life signals, I'm going to tell her to enable this part.